go.mod

```diff
@@ -2,6 +2,8 @@ go 1.18
 
 module github.com/irisnet/irismod
 
+require github.com/irisnet/irismod/nft v0.0.0-20230518034755-053ce7213e54
```

Comment:
How about we always use pseudo-version v0.0.0 rather than generated pseudo-version with commit hash?